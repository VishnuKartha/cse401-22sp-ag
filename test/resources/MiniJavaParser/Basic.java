class Foo {
    public static void main(String[] args) {
        a = a + 1;
    }
}