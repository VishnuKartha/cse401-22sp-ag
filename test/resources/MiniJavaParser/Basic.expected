Program
  MainClass Foo
    =
      a
      +
        a
        1
